apps/full-site-editing/full-site-editing-plugin/editor-site-launch/src/launch-modal/index.tsx

```diff
@@ -28,27 +29,44 @@ const LaunchModal: React.FunctionComponent< Props > = ( {
 	step,
 	isLaunching,
 } ) => {
+	const [ isOpen, setOpen ] = React.useState( true );
```

Comment:
Could you provide a little context on why this state is needed here?

There is already [visibility state for the modal](https://github.com/Automattic/wp-calypso/blob/449f385c51ac137a670433301b0cd8d1fd7ce6eb/apps/full-site-editing/full-site-editing-plugin/editor-site-launch/src/launch-button/index.tsx#L21) implemented in the component that renders the button + modal, as specified in [Modal docs](https://developer.wordpress.org/block-editor/components/modal/#usage-2).

Reply:
Oh this is because the WP's modal header is being replaced with our own modal header, and the close button on our own custom modal header needs a way to close the modal.

https://github.com/Automattic/wp-calypso/pull/44087/files/449f385c51ac137a670433301b0cd8d1fd7ce6eb#diff-2f6816d10f8e185d51e0b8937ad5a886R53-R57